x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -176,10 +176,7 @@ export class HeadlessChromiumDriverFactory {
       )(stderr$);
 
       const driver$ = Rx.of(
-        new HeadlessChromiumDriver(page, {
-          logger: this.logger,
-          inspect: this.browserConfig.inspect,
-        })
+        new HeadlessChromiumDriver(page, { inspect: this.browserConfig.inspect })
```

Comment:
In order to get control over how things get logged from within the Chromium driver, I decided it's best to NOT pass the logger to the `HeadlessChromiumDriver` constructor. Logging happens via the caller's `logger` object that gets passed to each method of the driver.